pychemy/peptide_sets.py

```diff
@@ -106,3 +106,178 @@ def balanced_sets(proteins, max_set_size = 10, unique = 1):
      sets.append([p])
   return sets
 
+##################################################################################
+# Calculate ionizability of peptides based on the method used by STEPP from PNNL #
+##################################################################################
+
+from pychemy.peptides import mass_from_sequence
+import numpy as np
+import csv
+
+NUM_FEATURES = 35
+
+AA_order = ['A', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'K', 'L', 'M', 'N', 'P', 'Q', 'R', 'S', 'T', 'V', 'W', 'Y']
+
+with open("resources/STEPP/STEPP_NormFactor_mean.txt") as tsv:
+  STEPP_mean = np.array([float(line[0]) for line in csv.reader(tsv, delimiter="\t")])
+
+with open("resources/STEPP/STEPP_NormFactor_std.txt") as tsv:
+    STEPP_std = np.array([float(line[0]) for line in csv.reader(tsv, delimiter="\t")])
+        
+with open("resources/STEPP/STEPP_Weights.txt") as tsv:
+  STEPP_weights = np.array([float(line[0]) for line in csv.reader(tsv, delimiter="\t")])   
+  
+non_polar_hydrophobic = ['A','F','G','I','L','M','P','V','W','Y']
```

Comment:
these should probably just be sets, not lists, since order does not seem to matter here.